VersionManager.py

```diff
@@ -131,6 +131,7 @@ def replace_version(version_obj):
     print('Updating source files to version {0}\n'.format(newver))
     for myfile in GREGORIO_FILES:
         result = []
+        following_file = False
```

Comment:
Originally I used just `following` but thought that I should indicate that we're expecting a file name format version substitution in the next line, hence the added `_file` (in case at some point a similar solution has to be implemented for a regular version number substitution).  Perhaps a variable name even more descriptive would be better, like: `following_line_filename`